openam-federation/openam-federation-library/src/main/java/com/sun/identity/saml2/meta/SAML2MetaSecurityUtils.java

```diff
@@ -230,19 +230,18 @@ public static Document sign(String realm, EntityDescriptorElement descriptor)
     }
 
     /**
-     * Verifies signatures in entity descriptor represented by the 
+     * Verifies signatures in entity descriptor represented by the
      * <code>Document</code>.
      * @param doc The document.
-     * @throws SAML2MetaException if unable to verify the entity descriptor. 
+     * @throws SAML2MetaException if unable to verify the entity descriptor.
      */
     public static void verifySignature(Document doc)
         throws SAML2MetaException
     {
         NodeList sigElements = null;
         try {
-            Element nscontext =
-                    org.apache.xml.security.utils.XMLUtils
-                            .createDSctx (doc,"ds", Constants.SignatureSpecNS);
+            Element nscontext = com.sun.identity.saml2.common.XmlUtils.createNamespaceElement(
```

Comment:
Isn't this what the new XmlUtils was created for?

Reply:
I do not understand what you mean. I have created XmlUtils with one method removed from the external library (after upgrading to the latest version). I was not able to use another existing method. So I also used it in this place.